ruby/build_jruby_release.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# This file should be executed with jruby
+
+set -ex
+
+# --- begin runfiles.bash initialization ---
+# Copy-pasted from Bazel's Bash runfiles library (tools/bash/runfiles/runfiles.bash).
+set -euo pipefail
+if [[ ! -d "${RUNFILES_DIR:-/dev/null}" && ! -f "${RUNFILES_MANIFEST_FILE:-/dev/null}" ]]; then
+  if [[ -f "$0.runfiles_manifest" ]]; then
+    export RUNFILES_MANIFEST_FILE="$0.runfiles_manifest"
+  elif [[ -f "$0.runfiles/MANIFEST" ]]; then
+    export RUNFILES_MANIFEST_FILE="$0.runfiles/MANIFEST"
+  elif [[ -f "$0.runfiles/bazel_tools/tools/bash/runfiles/runfiles.bash" ]]; then
+    export RUNFILES_DIR="$0.runfiles"
+  fi
+fi
+if [[ -f "${RUNFILES_DIR:-/dev/null}/bazel_tools/tools/bash/runfiles/runfiles.bash" ]]; then
+  source "${RUNFILES_DIR}/bazel_tools/tools/bash/runfiles/runfiles.bash"
+elif [[ -f "${RUNFILES_MANIFEST_FILE:-/dev/null}" ]]; then
+    source "$(grep -m1 "^bazel_tools/tools/bash/runfiles/runfiles.bash " \
+              "$RUNFILES_MANIFEST_FILE" | cut -d ' ' -f 2-)"
+else
+echo >&2 "ERROR: cannot find @bazel_tools//tools/bash/runfiles:runfiles.bash"
+exit 1
+fi
+# --- end runfiles.bash initialization ---
```

Comment:
optional: This boilerplate is why I created inline_sh_binary.  I don't have a strong preference here, but if you can get this smaller it might be nice to define it inline in the BUILD.bazel file

Reply:
This code seems like a pretty common pattern in many bazel files using `sh_binary` and I don't think we can cut it down much and putting it in the BUILD.bazel file as is seems clunky.